config_utilities/include/config_utilities/config.h

```diff
@@ -255,4 +257,42 @@ inline void check(Ts... args) {
   internal::Visitor::visitCheck(CheckType(std::forward<Ts>(args)...));
 }
 
+/**
```

Comment:
These functionalities are probably best going into a separate file (maybe `updates.h` or so?) We've been trying to keep includes minimal and `config.h` contains everything needed to add a `declare_config` call (so the rest of the config-user does not need to know about all the under the hood workings.